src/main/java/com/targomo/client/api/TravelOptions.java

```diff
@@ -505,7 +518,6 @@ public boolean equals(Object o) {
                 Objects.equals(targetGeohashes, that.targetAddresses) &&
                 Objects.equals(rushHour, that.rushHour) &&
                 Objects.equals(travelTimes, that.travelTimes) &&
-                travelType == that.travelType &&
```

Comment:
`travelTypes` should be still included in the `equals` and `hashCode` methods, no?

Reply:
Its already there, because there was already a `Set<TravelType> travelTypes`. That one was only used for statistics batch tool, I moved it there (`BatchTravelOptions`).